src/app/devices/deviceContent/components/deviceIdentity/deviceIdentity.tsx

```diff
@@ -203,29 +206,20 @@ export default class DeviceIdentityInformation
             >
                 <div className="sas-token-section">
                     <Dropdown
+                        className={'sas-token-key-field'}
                         label={context.t(ResourceKeys.deviceIdentity.authenticationType.sasToken.symmetricKey)}
                         selectedKey={sasTokenSelectedKey || undefined}
                         options={options}
-                        styles={{
-                            dropdown: {
-                                marginBottom: 5,
-                                width: 300
-                            }
-                        }}
                         onChange={this.onSelectedKeyChanged}
                     />
                     <SpinButton
+                        className={'sas-token-expiration-field'}
                         label={context.t(ResourceKeys.deviceIdentity.authenticationType.sasToken.expiration)}
                         labelPosition={position}
                         min={0}
                         max={Number.MAX_SAFE_INTEGER}
                         onBlur={this.onExpirationChanged}
-                        styles={{
-                            root: {
-                                marginBottom: 5,
-                                width: 300
-                            }
-                        }}
+                        defaultValue={`${defaultExpiration}`}
```

Comment:
if you use 'value' prop instead of 'defaultValue', can we get rid of sasTokenExpiration  from the state? i don't see it being used in the render here.